Physics/src/pointmass.cpp

```diff
@@ -49,6 +29,111 @@ void PointMass::integrate(real duration) {
 	clearAccumulator();
 }
 
+real PointMass::getKineticEnergy() {
+	return 0.5 * (1.0f / inverseMass) * velocity.magnitude();
+}
+
+// ------------ Mass ------------
+
+real PointMass::getMass() const
+{
+	if (PointMass::inverseMass <= 0.0)
+	{
+		return REAL_MAX;
+	}
+	else
+	{
+		return ((real)1.0) / inverseMass;
+	}
+}
+
+void PointMass::setMass(real mass) {
+	inverseMass = (real)1.0 / mass;
```

Comment:
Nitpick: Wrap the cast in brackets like above

Reply:
Resolved